modules/files.nix

```diff
@@ -102,10 +102,10 @@ in
       default = { };
       example = lib.literalExpression ''
         {
-          "ftplugin/nix.lua".text = '''
-            vim.opt.tabstop = 2
-            vim.opt.shiftwidth = 2
-            vim.opt.expandtab = true
+          "after/ftplugin/nix.lua".text = '''
```

Comment:
I think the original intent of this example was to show a simple file being added. Moving the file to `after` makes the example more complex, IMO.

I assume the reason you've done this is because this specific example of setting options without using `after` was causing you issues in some specific scenario? E.g. a scenario where you set the same options in your main nixvim config?

I wonder it it'd be better to instead come up with a different example, or to have an example that writes both `after` and non-`after` files?

Reply:
Yes, I'm aware of what `after` is.